style.css

```diff
@@ -10,28 +10,31 @@
 
 .left {
 float: left;
-width: 50%;
+width: 40%;
 height: 100%;
 color: red;
+font-size: 14px;
```

Comment:
it is a good idea to set font-size for body, and remove it from p tag. So it will mean that by default your page is using 14px font-size